src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -1029,11 +1029,10 @@ private function addValidationSection(ArrayNodeDefinition $rootNode, callable $e
                     if (isset($v['enable_annotations'])) {
                         trigger_deprecation('symfony/framework-bundle', '6.4', 'Option "enable_annotations" at "framework.validation" is deprecated. Use the "enable_attributes" option instead.');
 
-                        if (!isset($v['enable_attributes'])) {
-                            $v['enable_attributes'] = $v['enable_annotations'];
-                        } else {
+                        if (isset($v['enable_attributes'])) {
                             throw new LogicException('The "enable_annotations" and "enable_attributes" options at path "framework.validation" must not be both set. Only the "enable_attributes" option must be used.');
```

Comment:
should we even forbid having both ? Does `enable_attributes` also enable the support for annotations ?

Reply:
IMHO having both should be forbidden. `enable_attributes` is new, so if one use it, they must remove the value of `enable_annotations`

Reply:
having both _in the same file_ might be forbidden. but here, you check that after the merge of files, not when normalizing each file.